src/app/tests/suites/certification/PICS.yaml

```diff
@@ -2748,6 +2748,52 @@ PICS:
           and is ReadOnly"
       id: DRLK.S.A002c.ReadOnly
 
+    - label:
+          "Does the DUT(server) support the AliroReaderVerificationKey attribute
+          and is ReadOnly"
+      id: DRLK.S.A0080.ReadOnly
```

Comment:
These are not valid PICS, right?

Reply:
removed these PICS as they are not being used